src/http/logical.rs

```diff
@@ -55,8 +55,16 @@ async fn logical_request_handler(
     let conn = req.conn_data::<Connection>().unwrap();
     log::debug!("logical request, connection info: {:?}, method: {:?}, path: {:?}", conn, method, path);
 
+    let mut req_conn = ReqConnection::default();
+    req_conn.peer_addr = conn.peer.to_string();
+    if conn.tls.is_some() {
+        let tls_client_info = conn.tls.as_ref().unwrap();
```

Comment:
The variable `tls_client_info` is evitable:

~~~
req_conn.peer_tls_cert = conn.tls.as_ref().unwrap().client_cert_chain.clone();
~~~

Reply:
> The variable `tls_client_info` is evitable:
> 
> ```
> req_conn.peer_tls_cert = conn.tls.as_ref().unwrap().client_cert_chain.clone();
> ```

fixed